pkg/common/bundleutil/bundle.go

```diff
@@ -101,6 +102,66 @@ func CommonBundleFromProto(b *types.Bundle) (*common.Bundle, error) {
 	}, nil
 }
 
+// SPIFFEBundleToBundleUtil is a temporary function that converts a spiffebundle.Bundle to bundleutil.Bundle. This
+// function should be used only for restricting the scope of the changes in places that still use bundleutil.Bundle.
+// It should be removed as soon as we don't have any other reference to bundleutil.Bundle.
```

Comment:
nit: Adding the word "TODO" here may help to highlight the fact that removing this is something pending.
It will also help to find it when searching for "TODO" things.